python/setup.py

```diff
@@ -291,7 +291,7 @@ def get_packages(self):
         "click >= 7.0, <= 8.0.4",
         "dataclasses; python_version < '3.7'",
         "filelock",
-        "grpcio >= 1.28.1, <= 1.43.0; python_version < '3.10'",
+        "grpcio >= 1.32.0, <= 1.43.0; python_version < '3.10'",
```

Comment:
Should we upgrade it in requirements.txt as well? The comment says "If you change it, please reflect the change in the matching section of requirements/requirements.txt".